### ISSUE TITLE
How and when to use size_frac?

### ISSUE THREAD

**Issue by turbomam** [microbiomedata/sheets_and_friends#95]
I think we omitted slot `size_frac` from the DH interfaces for soil samples. I also think I had been including them in the initial DH interfaces for all other environmental packages, but then it seems to have disappeared at some point. I'm adding it back in for all non-soil DH interfaces.

`size_frac` is used in the following subclasses of `MIMS`. (That's probably all of the subclasses!)

- MIMS
- agriculture MIMS
- air MIMS
- built environment MIMS
- food-animal and animal feed MIMS
- food-farm environment MIMS
- food-food production facility MIMS
- food-human foods MIMS
- host-associated MIMS
- human-associated MIMS
- human-gut MIMS
- human-oral MIMS
- human-skin MIMS
- human-vaginal MIMS
- hydrocarbon resources-cores MIMS
- hydrocarbon resources-fluids_swabs MIMS
- microbial mat_biofilm MIMS
- miscellaneous natural or artificial environment MIMS
- plant-associated MIMS
- sediment MIMS
- soil MIMS
- symbiont-associated MIMS
- wastewater_sludge MIMS
- water MIMS

**Comment by mslarae13** [comment]
Correct. The definition was ambiguous for soil. We replaced it with a different term. I expect it will be replaced for the other packages too as we go through the manual curation & additions. For now, it can stay as is.

**Comment by turbomam** [comment]
> Correct. The definition was ambiguous for soil. We replaced it with a different term. I expect it will be replaced for the other packages too as we go through the manual curation & additions. For now, it can stay as is.

OK, thanks. It will be easy to remove `size_frac` from any DH interface at any time, and only slightly harder to reassign more appropriate slots for the general theme of fractionating mixtures based on size.
